mesos.yml

```diff
@@ -0,0 +1,310 @@
+        echo "server.1=$IPVAR1:2888:3888" > /etc/zookeeper/conf/zoo.cfg
+        echo "dataDir=/var/zookeeper" >> /etc/zookeeper/conf/zoo.cfg
+        echo "clientPort=2181" >> /etc/zookeeper/conf/zoo.cfg
+
+
+        #Quorum Value Defaul=1
+        #echo "1" > /etc/mesos-master/quorum
+        echo $IPVAR1 > /etc/mesos-master/ip
+        echo $IPVAR1 > /etc/mesos-master/hostname
+        mkdir -p /etc/marathon/conf
+        cp /etc/mesos-master/hostname /etc/marathon/conf
+        cp /etc/mesos/zk /etc/marathon/conf/master
+        sed 's/mesos/marathon/' /etc/marathon/conf/master > /etc/marathon/conf/zk
+        echo 'docker,mesos' > /etc/mesos-slave/containerizers
+        echo '5mins' > /etc/mesos-slave/executor_registration_timeout
+        stop mesos-slave
+        sleep 10
+        echo "manual" | sudo tee /etc/init/mesos-slave.override
+        stop zookeeper
+        sleep 10
+        start zookeeper
+        sleep 10
+        start mesos-master
+        sleep 10
+        start marathon
+        
+        apt-get install -y nginx
+        rm /etc/nginx/sites-enabled/default
+        echo "server { listen 80; server_name ${var.koding_user_username}.mesosphere.live; location / { proxy_pass http://$IPVAR1:8080; }}" >/etc/nginx/sites-enabled/marathon
+        echo "server { listen 80; server_name ${var.koding_user_username}-mesos.mesosphere.live; location / { proxy_pass http://$IPVAR1:5050; }}" >/etc/nginx/sites-enabled/mesos
+        service nginx restart
+        
+        PUBLIC_IP=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
+        
+        wget https://github.com/mesosphere/mesos-dns/releases/download/v0.5.2/mesos-dns-v0.5.2-linux-amd64
+        mv mesos-dns-v0.5.2-linux-amd64 /usr/local/sbin/mesos-dns
+        chmod +x /usr/local/sbin/mesos-dns 
+        
+        cat <<EOT >> /etc/mesos-dns-config.json
+        {
+          "zk": "zk://$IPVAR1:2181/mesos",
+          "masters": ["$IPVAR1:5050"],
+          "refreshSeconds": 60,
+          "ttl": 60,
+          "domain": "mesos",
+          "port": 53,
+          "resolvers": ["8.8.8.8"],
+          "timeout": 5, 
+          "httpon": true,
+          "dnson": true,
+          "httpport": 8123,
+          "externalon": true,
+          "listener": "0.0.0.0",
+          "SOAMname": "ns1.mesos",
+          "SOARname": "root.ns1.mesos",
+          "SOARefresh": 60,
+          "SOARetry":   600,
+          "SOAExpire":  86400,
+          "SOAMinttl": 60,
+          "IPSources": ["mesos", "host"]
+        }
+        EOT
+        mesos-dns -config=/etc/mesos-dns-config.json & 
+        sed -i "s/nameserver 10.0.0.2/nameserver $IPVAR1\nnameserver 10.0.0.2/" /etc/resolv.conf
+        echo "You can access the Marathon Web UI at http://${var.koding_user_username}.mesosphere.live"
+        echo ""
+        echo "You can access the Mesos Web UI at http://${var.koding_user_username}-mesos.mesosphere.live"
+        
+        
+    slave_1:
```

Comment:
Instead of duplicating slave definition we could have only 1 slave and control the number of them with Terraform's `count` property.